provisioning/ci/aws.env

```diff
@@ -0,0 +1,2 @@
+AWS_REGION=eu-central-1
+AWS_ROLE_TO_ASSUME=arn:aws:iam::480319613404:role/ci-job-runner-terraform-GithubActionsRole-10VD8SPVROKFR
```

Comment:
Nesedí mí název té role :) GA tady nepoužíváme, vzniklo to tím jak jsi to kopíroval od jinud :)

Reply:
Jako ze ti vadi, ze to tu je nebo ti neco nesedi na te konkretni hodnote?

Reply:
Jako že se to jmenuje `GithubActionsRole`

Reply:
No ten nazev je spravne. Je to tu "zbytecne" zkopirovane odjinud (`oauth-service`). Kdyz uz jsem ale kopiroval, prislo mi dobry to nechat, kdyz je to funkcni. Pro pripad, ze by tu nekdo pozdeji chtel nahodit GHA (driv nebo pozdeji se tomu nevyhneme). Rad bych to mel soucasti "sablony".

Reply:
No já souhlasím s Erikem, je to dost napiču, funkční to není (protože se to nepoužívá), jestli se za rok dostaneme k přesunu job runner na github actions (což pořád nějak skrytě doufám, že nenastane), tak to stejně bude už zase nějak jinak a akorát to tady do té dobry bude dělat chaos - to platí i pro aws.yaml a spol

Reply:
https://github.com/keboola/job-runner/pull/239